yarn.lock

```diff
@@ -20868,10 +20868,10 @@ react-dev-utils@^11.0.3:
     strip-ansi "6.0.0"
     text-table "0.2.0"
 
-react-docgen-typescript@^1.16.6, react-docgen-typescript@^1.22.0:
-  version "1.22.0"
-  resolved "https://registry.yarnpkg.com/react-docgen-typescript/-/react-docgen-typescript-1.22.0.tgz#00232c8e8e47f4437cac133b879b3e9437284bee"
-  integrity sha512-MPLbF8vzRwAG3GcjdL+OHQlhgtWsLTXs+7uJiHfEeT3Ur7IsZaNYqRTLQ9sj2nB6M6jylcPCeCmH7qbszJmecg==
+react-docgen-typescript@^2.0.0, react-docgen-typescript@^2.1.0:
+  version "2.1.1"
```

Comment:
This is not the latest version, because that one has a bug with how it determines the parent filename a prop came, which prevents the conformance tests from properly ignoring props that came from `@types/react` (I'm planning to make a PR there with a fix, but that may get hung up on discussion because the "proper" behavior hasn't been well-defined)

Reply:
https://github.com/styleguidist/react-docgen-typescript/pull/405